docker-compose.dev.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.7'
+networks:
+  rs-net:
+    driver: bridge
+services:
+  rs:
+    image: iudx/rs-dev:latest
+    volumes:
+      - ./secrets/credentials/all-verticles-configs/config-dev.json:/usr/share/app/secrets/credentials/all-verticles-configs/config.json
```

Comment:
Unless secretes is a local folder as well..?
In that case have a "secrets" folder on the repo with a .gitkeep file

Reply:
secrets folder is a local folder and don't want to be git versioned!!!, and hence used .gitignore for it. won't using .gitkeep would version the directory?

Reply:
Yes we'll need the directory to indicate people where to put the configs. 
In gitignore you should ignore the conf files while retaining the folder structure.

Reply:
Okay will retain old configs directory with structure:
```sh
secrets/
├── all-verticles-configs
│   ├── config-depl.json
│   ├── config-dev.json
│   └── config-x.json
├── keystore.jks
└── one-verticle-configs
```
Cannot use existing configs directory, because the files inside it is committed and everyone needs to do `git rm --cached` to ignore the files configs. See [here](https://stackoverflow.com/questions/6535362/gitignore-after-commit)